Jenkinsfile

```diff
@@ -124,15 +124,20 @@ pipeline {
             steps {
                 withMaven(maven: 'maven-latest', jdk: 'jdk8-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}') {
                     script {
-                        // If this build is not a pull request, run full owasp scan. Otherwise run incremental scan
+                        // If this build is not a pull request, run owasp scan on the distribution
                         if (env.CHANGE_ID == null) {
-                            sh 'mvn org.commonjava.maven.plugins:directory-maven-plugin:highest-basedir@directories dependency-check:check dependency-check:aggregate -q -B -Powasp -DskipTests=true -DskipStatic=true -pl !$DOCS $DISABLE_DOWNLOAD_PROGRESS_OPTS'
+                            sh 'mvn dependency-check:aggregate -q -B -Powasp-dist -pl !$DOCS -P !itests $DISABLE_DOWNLOAD_PROGRESS_OPTS'
```

Comment:
❔ Why was the highest-basedir@directories removed? Is it because this is now more on the distribution and we can rely on a relative path to the `dependency-check-maven-config.xml` file?

Reply:
That plugin allowed us to run the dependency-check from submodules and still use the suppression file at the root. That was useful because `dependency-check:check` ran on every module. The `dependency-check:aggregate` goal only runs on the root module, so we don't need to do this.